packages/brick_offline_first/test/offline_queue/request_sqlite_cache_manager_test.dart

```diff
@@ -78,18 +78,18 @@ void main() {
         final request = await requestManager.prepareNextRequestToProcess();
         expect(request?.method, 'POST');
 
-        final asCacheItem = RequestSqliteCache(request!);
+        final asCacheItem = RestRequestSqliteCache(request!);
         await asCacheItem.insertOrUpdate(await requestManager.getDb());
         // Do not retry request if the row is locked and serial processing is active
         final req = await requestManager.prepareNextRequestToProcess();
-        expect(req, isNull);
+        expect(req, isNotNull);
```

Comment:
This test shouldn't change
```suggestion
        expect(req, isNull);
```